sklearn/model_selection/tests/test_validation.py

```diff
@@ -670,6 +670,31 @@ def test_learning_curve_batch_and_incremental_learning_are_equal():
                               test_scores_batch.mean(axis=1))
 
 
+def test_learning_curve_batch_and_incremental_shuffle():
```

Comment:
Since this test does nothing but check that `shuffle=True` gives results akin to `shuffle=False` (rather than ensuring that it's shuffled), I think you should just add `for shuffle in [True, False]` to tests above, rather than create a new smoke test.